modules/frontend/searchsharding.go

```diff
@@ -70,7 +70,7 @@ func (r *searchResponse) addResponse(res *tempopb.SearchResponse) {
 	defer r.mtx.Unlock()
 
 	r.results.Traces = append(r.results.Traces, res.Traces...)
-	r.results.Metrics.InspectedBlocks += res.Metrics.InspectedBlocks
+	// purposefully ignoring InspectedBlocks as that value is set by the sharder
```

Comment:
I expected to see something like this in the frontend code
```
if queryparams contains "backend_search" {
  // call the cloud function here
  .
  http.get(frontend.cfg.cloud_function_endpoint)
  .
}
```

is that integrated yet or will be done next?

Reply:
I will be doing that next. The next PR is going to focus on:
- adding all config options (including option to add a serverless endpoint)
- testing
- collapsing the backend search query path into the standard query path